lib/binding/kvdb_interface.c

```diff
@@ -440,6 +436,10 @@ hse_init(const char *const config, const size_t paramc, const char *const *const
         }
     }
 
+    err = ikvdb_init();
+    if (err)
+        goto out;
+
```

Comment:
We used to have a problem where the rest server could call into hse while hse was still setting things up and it would crash.  What prevents that now?

Reply:
I guess nothing, but I am pretty sure this out of order operation is causing an abort in mongo js tests, since before if the rest server failed, nothing from ikvdb_init() would get cleaned up. ikvdb_init() is a source of the same issues that we had with platform_init() recently.

Reply:
Can we init the rest server early, and make it live as the last step in hse_init?

Reply:
Right, just don't publish the endpoints until everything's ready to go, and remove them before you tear down the subsystem that they call into (I think we do the latter in most/all places?)